app/views/home/maintainers.md

```diff
@@ -0,0 +1,45 @@
+> Audience: "developers", meaning anyone who has skills that are relevant to software development, that identify as a "maintainer" regularly contribute to an open source project.
+
+> What are you asking me to do?
+
+Take one day this week to make your project easier to use and more welcoming to new contributors.
+
+> Why should I care?
+
+You could use help
+
+> What are the reasons I should care about getting more contributors for my open source project?
+
+- **Reduce your workload** - Regular contributors help
+- **Grow your skills and help others** - Whatever your skill level, working on open source software is a great way to learn skills like leadership, mentoring, and project management.
+- **Build your reputation** -
+- **Create better software** - people with different skills and passions working together create software that
+
+> How should I get started?
+
+## Join the community on Open Source Friday
+
+> Logistics: remove any barriers to making it happen this Friday
+
+### Set aside time
+
+TODO: words about what might block you (time, boss, etc)
+
+CTA: Add to calendar
+
+> Knowledge: What if I feel like I don't know enough yet?
+
+## Learn how to build a welcoming community
+
+Building a community that encourages people to use, contribute to, and evangelize your project.
+
+CTA: [Read the guide](https://opensource.guide/building-community/)
+
+> Onramp: I'm scared, can you hold my hand?
+
+## ????
+
+- ensure your project's README file details whether your project wants contributions, the project's scope and how to contribute to it.
+- improve onboarding docs (missing CONTRIBUTING, CODE_OF_CONDUCT, etc.)
+- diagnose community problems? (e.g. fossheartbeat)
+- create first timer issues
```

Comment:
Make documentation for specific contribution cases to make it possible for people to come in and look for how to do exactly what they're looking for, and ignore irrelevant information (e.g. on Exercism that would be: fix a typo in an exercise README, or add a new exercise, or port an existing exercise to a new programming language, launch a new language track, extract canonical test inputs/outputs for an exercise).